src/Options/Definitions.js

```diff
@@ -533,6 +539,12 @@ module.exports.ParseServerOptions = {
     help: 'Starts the liveQuery server',
     action: parsers.booleanParser,
   },
+  transformEmailAndUsernameToLowerCase: {
```

Comment:
I'm not sure whether it's good to combine this into 1 option. It may be beneficial (for other use cases) to introduce 2 separate options that allow developers to control that for email and username separately.

Reply:
I understand, but these options will live thought out Parse Server, not just for this specific user case. So this "package deal" may be a "no deal" for other use cases. If we decide to change divide the later on it will be a breaking change and more complicated, so I'd suggest to separate them now.

Reply:
Hi @mtrezza  @b10f , following YAGNI and KISS, currently there is no a real need to split the option. As @b10f it's really specific to DBs not supporting collation. Here i'm in favor to the bundle, and if someone need separate usage it could be a another PR.

Reply:
Changes need to be considered from a broader perspective, not just for a single use-case. We have seen this over and over again in the past where combined features need to be divided, causing breaking changes which we generally want to avoid. Converting only emails or only usernames to lowercase are valid use cases. The time spent in this conversation is more than would have been needed to split the single option into 2, which takes maybe 5 minutes.